pilot/pkg/serviceregistry/federation/controller.go

```diff
@@ -491,27 +491,31 @@ func (c *Controller) updateGateways(serviceList *federationmodel.ServiceListMess
 }
 
 func (c *Controller) getEgressServiceAddrs() ([]model.NetworkGateway, []string) {
-	endpoints, err := common.EndpointsForService(c.kubeClient, c.egressName, c.namespace)
+	endpointSlices, err := common.EndpointSlicesForService(c.kubeClient, c.egressName, c.namespace)
 	if err != nil {
-		c.logger.Errorf("unabled to retrieve endpoints for federation egress gateway %s: %s", c.egressName, err)
+		c.logger.Errorf("failed to retrieve EndpointSlices for federation egress gateway %s: %s", c.egressName, err)
```

Comment:
I thought that "failed" is more common phrase than "unable" when it comes to grepping logs. WDYT?